src/App.vue

```diff
@@ -3,7 +3,7 @@
     <UiLoading v-if="app.loading || !app.init" class="overlay big" />
     <div v-else>
       <Topnav />
-      <div class="pb-6 overflow-hidden">
+      <div class="pb-6 overflow-hidden" style="min-height: 420px;">
```

Comment:
What's the need of this?

Reply:
![image](https://user-images.githubusercontent.com/15967809/111899132-9cf12c00-8a50-11eb-9b72-2d24081981f6.png)

Without this .. when there are no results in the page dropdown goes like this

Reply:
Then it's better to have this div taking the full height of the page instead of chosing a height of 420px